src/js/themes/__tests__/__snapshots__/theme-test.js.snap

```diff
@@ -1552,8 +1551,7 @@ exports[`Grommet hpe theme 1`] = `
   display: flex;
   box-sizing: border-box;
   max-width: 100%;
-  background-color: #FD6FFF;
-  color: #333333;
+  background: accent-2;
```

Comment:
Something doesn't look right to have "accent-2" in the CSS.

Reply:
Agreed, we shouldn't have accent or neutral colors on the hpe theme test.
I've created an array of colors that are more specific to the hpe theme and make the test more efficient.